### ISSUE TITLE
Doesn't seem to work with module marked as external

### ISSUE THREAD

**Issue by Narretz** [dot-build/rollup-plugin-includepaths#10]
So I was under the impression that doing

`import {foo} from 'foo'`

with 
`include: {foo: 'node_modules/foo.js' }`

would use the file under node_modules as  the import "base".
Instead I always get "Treating 'foo' as external dependency" and it never gets included. Is this expected?

**Comment by adriengibrat** [comment]
same here, option.include seems broken since #3 has changed cache key structure...

**Comment by darlanalves** [comment]
Hello guys! Sorry for the super late response.

So, from what I know so far about rollup, it goes through the list of modules included and pass their names down the chain of plugins to be resolved.

@Narretz:
The purpose of this plugin is to simplify the inclusion of modules from paths relative to one or more base paths. In your case, it would _not_ be a relative resolution, but a node module instead. If you don't use another plugin to include node_modules folder, it will be marked as external and left out of the final bundle.
That being said, the _include_ option is a bypass of the entire plugin code, pointing a module name directly to a file path. I don't know exactly why your module is being marked as external, it should be included in the bundle as well.
I think a better approach in this case would be a removal of your _include_ option here and add the [https://github.com/rollup/rollup-plugin-commonjs](commonjs) plugin in your setup to add node modules.

@adriengibrat:
Thanks for your PR. I'm adding it to the plugin 👍

**Comment by adriengibrat** [comment]
Great, thanks for the merge!

Could we expect a release including the bug fix ?
I hope a 0.1.6, unless you have more to add.

P.S. I've payed lately with npm commands, i discovered you can :
- hook in [`npm version`](https://docs.npmjs.com/cli/version) with scripts in `package.json` since npm@2.13.2 :
```json
"engines": {
  "npm": ">= 2.13.2"
},
"scripts": {
  "postversion": "git push && git push --tags"
},
```
- pass commit message to `npm version`
```shell
npm version patch -m "%s include option won't treat included dependencies as external dependency"
```
- [`npm deprecate`](https://docs.npmjs.com/cli/deprecate) bugged releases:
```shell
npm deprecate rollup-plugin-includepaths@"0.1.3 - 0.1.5" "v0.1.6 fix issue with include option"
```
- make range expression like a pro with https://semver.npmjs.com ;)

**Comment by darlanalves** [comment]
Fixed with release 0.1.6 (#11). Closing the issue.
